docs/poc2023q1/mailbox-controller.md

```diff
@@ -48,9 +52,39 @@ I0127 00:21:48.876022   24503 main.go:206] "Found APIExport view" exportName="wo
 I0127 00:21:48.877965   24503 shared_informer.go:255] Waiting for caches to sync for mailbox-controller
 I0127 00:21:48.978352   24503 shared_informer.go:262] Caches are synced for mailbox-controller
 I0127 00:21:48.978414   24503 main.go:169] "Informers synced"
+I0303 16:45:52.528677   62289 main.go:113] "Command line flag" add_dir_header="false"
+I0303 16:45:52.528859   62289 main.go:113] "Command line flag" alsologtostderr="false"
+I0303 16:45:52.528865   62289 main.go:113] "Command line flag" concurrency="4"
+I0303 16:45:52.528869   62289 main.go:113] "Command line flag" inventory-context="root"
+I0303 16:45:52.528872   62289 main.go:113] "Command line flag" inventory-kubeconfig=""
+I0303 16:45:52.528876   62289 main.go:113] "Command line flag" log_backtrace_at=":0"
+I0303 16:45:52.528880   62289 main.go:113] "Command line flag" log_dir=""
+I0303 16:45:52.528883   62289 main.go:113] "Command line flag" log_file=""
+I0303 16:45:52.528886   62289 main.go:113] "Command line flag" log_file_max_size="1800"
+I0303 16:45:52.528890   62289 main.go:113] "Command line flag" logtostderr="true"
+I0303 16:45:52.528893   62289 main.go:113] "Command line flag" one_output="false"
+I0303 16:45:52.528897   62289 main.go:113] "Command line flag" server-bind-address=":10203"
+I0303 16:45:52.528900   62289 main.go:113] "Command line flag" skip_headers="false"
+I0303 16:45:52.528904   62289 main.go:113] "Command line flag" skip_log_headers="false"
+I0303 16:45:52.528907   62289 main.go:113] "Command line flag" stderrthreshold="2"
+I0303 16:45:52.528911   62289 main.go:113] "Command line flag" v="2"
+I0303 16:45:52.528914   62289 main.go:113] "Command line flag" vmodule=""
+I0303 16:45:52.528918   62289 main.go:113] "Command line flag" workload-context=""
+I0303 16:45:52.528921   62289 main.go:113] "Command line flag" workload-kubeconfig=""
+I0303 16:45:52.552450   62289 main.go:248] "Found APIExport view" exportName="workload.kcp.io" serverURL="https://192.168.58.123:6443/services/apiexport/root/workload.kcp.io"
+I0303 16:45:52.554741   62289 shared_informer.go:282] Waiting for caches to sync for mailbox-controller
+I0303 16:45:52.654988   62289 shared_informer.go:289] Caches are synced for mailbox-controller
+I0303 16:45:52.655018   62289 main.go:211] "Informers synced"
+```
+
+In a separate shell, make a workload management workspace as follows.
+
+```
+kubectl ws \~
+kubectl ws create work1 --enter
 ```
 
-In a separate shell, `kubectl create` the following object.
+Then in that workspace, `kubectl create` the following object.
```

Comment:
Maybe "the following SyncTarget object " ?

Reply:
done.